samples/bootloader/src/main.c

```diff
@@ -5,18 +5,16 @@
  */
 
 #include <zephyr/types.h>
+#include <errno.h>
 #include <toolchain.h>
 #include <misc/util.h>
+#include <misc/printk.h>
 #include <nrf.h>
-#include <errno.h>
 #include <pm_config.h>
 #include "bootloader.h"
-#include "bl_crypto.h"
-#include "fw_metadata.h"
-#include <misc/printk.h>
-
+#include <bl_crypto.h>
```

Comment:
What is the reason for this re-order? Doesn't seem much more ordered to me xD

Reply:
Haha :P.
- removed unneeded newlines
- misc includes put together
- errno further up (together with types)
- <> instead of ""